src/App.tsx

```diff
@@ -1,8 +1,9 @@
-import TaskListPage from './pages/task-list/TaskListPage'
-import './styles/reset.css'
+// import TaskListPage from './pages/task-list/TaskListPage'
+import { DialogExample } from "./DialogExample"
 
 function App() {
-  return <TaskListPage />
+  // return <TaskListPage />
+  return <DialogExample />
```

Comment:
@AppleLAN This is a demo, it will be removed before merging with the `main` branch.